test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex route
+   Given Minishift has state "Running"
+    When status code of HTTP request to access "ruby-ex" application url in namespace "ruby" is equal to "200"
+    Then body of HTTP request to access "ruby-ex" application url in namespace "ruby" contains "Welcome to your Ruby application on OpenShift"
+     And body of HTTP request to access "ruby-ex" application url in namespace "ruby" contains "<title>Welcome to OpenShift</title>"
```

Comment:
I think verifying "Welcome to your Ruby application on OpenShift" is more than enough. We are really not interested what exactly is displayed.